filterByCircularity: true
minCircularity: 0.74
filterByArea: true
minArea: 200
